src/App.vue

```diff
@@ -4,7 +4,12 @@
     <input v-model="inputValue" type="text" placeholder="Hello world">
   </div>
 
-  <Picker :native="false" @select="updateEmoji" />
+  <Picker
+      :native="false"
+      :search="true"
+      :groupIcons="true"
+      @select="updateEmoji"
+  />
```

Comment:
Since both options is true by default, I think we don't need to pass true props of those from App